evil-collection-integration.el

```diff
@@ -100,7 +100,7 @@
 
 ;;; Buffer-menu
 
-(evil-add-hjkl-bindings Buffer-menu-mode-map 'motion)
+(evil-add-hjkl-bindings Buffer-menu-mode-map '(normal motion)) ;;temporary?
```

Comment:
This line should be removed completely.

Reply:
I did that with no success, it still does not bind anything